pyinfra/facts/server.py

```diff
@@ -460,6 +468,7 @@ def process(self, output):
                     'groups': groups,
                     'uid': int(entries[2]),
                     'gid': int(entries[3]),
+                    'lastlog': login_time,
```

Comment:
As well as attaching the raw `lastlog` it might be nice to have parsed version of the date under key `login_time` or similar? ([could use dateutil similar to the `server.Date` fact](https://github.com/Fizzadar/pyinfra/blob/de8d4f3b364ea6ea7ece4841ebe97771bc002ae5/pyinfra/facts/server.py#L127)).

Reply:
I'm not sure what you mean by having a parsed version of the date for `login_time`.  The current format of the is straight from `lastlog` and would look something like
```
Sat Jun 19 14:18:07 -0400 2021
```
What change are you wanting specifically?  I would assume if a user wants something more specific, they can parse the output of the fact themselves.  I'm interested in hearing what you have in mind.

Reply:
Sorry that wasn't super clear, I think it's nice to provide the user a ready to go `datetime` object from the fact; thinking something like this:

```py
from dateutil.parser import parse as parse_date
...
{
    'lastlog': login_time,
    'login_time': parse_date(login_time),
}
```